src/actors/symbolication.rs

```diff
@@ -1315,30 +1318,23 @@ impl SymbolicationActor {
                 } else {
                     "unknown"
                 };
-                let kind = if perr.is_timeout() {
-                    SymbolicationError::Timeout
-                } else {
-                    SymbolicationError::Canceled
-                };
                 metric!(counter(metric) += 1, "reason" => reason);
-                if let SymbolicationError::Canceled = kind {
-                    Self::save_minidump(minidump, minidump_cache)
-                        .map_err(|e| log::error!("Failed to save minidump {:?}", &e))
-                        .map(|r| {
-                            if let Some(path) = r {
-                                sentry::configure_scope(|scope| {
-                                    scope.set_extra(
-                                        "crashed_minidump",
-                                        sentry::protocol::Value::String(
-                                            path.to_string_lossy().to_string(),
-                                        ),
-                                    );
-                                });
-                            }
-                        })
-                        .ok();
-                }
-                Err(kind)
+                Self::save_minidump(minidump, minidump_cache)
```

Comment:
Before, never saved the minidump for timeouts, and only for crashes, panics, and fatal errors in the child process.